examples/graph/deckgl-overlay.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import DeckGL, {OrthographicViewport} from 'deck.gl';
+import GraphLayoutLayer from './graph-layer/graph-layout-layer';
+
+export default class DeckGLOverlay extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {viewport, data} = this.props;
+
+    if (!data) {
+      return null;
+    }
+
+    const {width, height} = viewport;
+    const {layoutProps} = this.props;
+    const {layoutAccessors, linkAccessors, nodeAccessors, nodeIconAccessors} = this.props;
```

Comment:
I think `get*` is good for an individual accessor, but these are maps of accessors, not functions. "Get" is a verb and so implies a function, so it doesn't quite map right for these maps. If you insisted I'd change them but since you're fine with either, I'll leave as-is :)